configurations/local.template.ini

```diff
@@ -212,6 +212,9 @@ rtc_token_expiry  = 60
 ;indexing query max size before compressing
 indexing_query_compress_threshold = 819200
 
+authBrokerBaseUrl = @AUTH_BROKER_BASE_URL@
```

Comment:
seems like other params are divided with _ and not camel case so i would keep this convention